src/components/Editor/SearchBar.js

```diff
@@ -50,9 +51,9 @@ const SearchBar = React.createClass({
   getInitialState() {
     return {
       enabled: false,
-      functionSearchEnabled: false,
-      functionDeclarations: [],
-      functionSearchResults: [],
+      symbolSearchEnabled: false,
+      selectedSymbolType: "functions",
```

Comment:
Default to functions? I'm not sure if that's  what we want but seemed like a decent approach.